content/language/word-list.md

```diff
@@ -96,3 +96,29 @@ While master in and of itself is potentially neutral, the propensity in which it
 The above, as well as:
 
 * [github/renaming](https://github.com/github/renaming)
+
+## Bulletproof
+
+**Definition:** 
+Bulletproof is typically used in the context of cybersecurity to describe systems or applications that are relatively free from vulnerabilities.
+Bulletproof is also sometimes used in marketing communication when comparing products or services to those offered by competitors.
+
+**Proposed alternatives:** 
+
+In the cybersecurity context:
+* Secure
+* Safe
+* Protected
+
+In the marketing context:
```

Comment:
```suggestion
In the marketing context (except when describing something truly bulletproof):
```
I feel like this is a valuable point to make, as people may otherwise misinterpret the scope of this recommendation.

Reply:
I'm not sure, perhaps within the context of an initiative focused on word use in soft technologies no one is going to genuinely confuse the scope of the recommendation?

Maybe we can get even more general, to have just the security and the general:
```suggestion
In the general context:
```


Pulled out of that context, which I agree our terms ideally can be (either as is or with reasonable modifications), I'm feeling like anyone in the arms business already knows about their ability to use that term in a marketing context, and automatically its out of scope for every other reader.

Oh, and I'm not truly on a fence, I'm actually comfortably sitting in a chair writing this ...
/me ducks the incoming pillows from @Nytelife26